OpenNefia.Content/Resources/Locale/en_US/Elona/CharSheet.lua

```diff
@@ -0,0 +1,57 @@
+Elona.CharSheet = {
+    Potential = {
+        Superb = "Superb",
+        Great = "Great",
+        Good = "Good",
+        Bad = "Bad",
+        Hopeless = "Hopeless",
+    },
+    Name = "Name",
+    Aka = "Aka",
+    Race = "Race",
+    Sex = "Sex",
+    Class = "Class",
+    Age = "Age",
+    Height = "Height",
+    Cm = "cm",
+    Weight = "Weight",
+    Kg = "kg",
+    Level = "Level",
+    Exp = "EXP",
+    NextLv = "Next Lv",
+    God = "God",
+    Guild = "Guild",
+    Sanity = "Sanity",
+    Fame = "Fame",
+    Karma = "Karma",
+    Turns = "Turns",
+    TurnsPassed = "Turns",
+    Days = "Days",
+    DaysPassed = "Days",
+    Kills = "Kills",
+    Time = "Time",
+    CargoWt = "Cargo Wt",
+    CargoLmt = "Cargo Lmt",
+    EquipWt = "Equip Wt",
+    DeepestLv = "Deepest Lv",
+    DeepestLvDesc = function(level)
```

Comment:
This function I want to be moved into `SharedBuiltins` because I know it's going to be used in a couple of other places (like the ranking system). It should be called `ordinal(number)`.

Reply:
changed in 2e3ad22